### PR TITLE
EBODAC-511 - Adding new fixes for the last issues (16-08-16)

### PR DESCRIPTION
**Pull request by rubenmartin**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ParticipantEditPage.java [hunk 1]
```diff
@@ -22,7 +22,8 @@
     private Map<String, String> mapLangPos = new HashMap<String, String>();
 
     // Object initialization for log
-    //private static Logger log = Logger.getLogger(ParticipantEditPage.class.getName());
+    // private static Logger log =
+    // Logger.getLogger(ParticipantEditPage.class.getName());
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ParticipantEditPage.java]: remove commented out code

Comment (rubenmartin) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/ParticipantEditPage.java]: @pmuchowski  I am surprise with the changes, I have added all the fixes, thanks for your input.

**Comment by pmuchowski** [comment]
test this please

**Comment by rubenmartin** [comment]
Thanks

**Pull request merged**